promptsource/promptsource.py

```diff
@@ -157,12 +157,14 @@ def save_data(message="Done!"):
     st.sidebar.subheader("Dataset Schema")
     st.sidebar.write(render_features(dataset.features))
 
-    
-    st.sidebar.subheader("Select Example")
-    example_index = st.sidebar.slider("Select the example index", 0, len(dataset)-1)
-    
-    example = dataset[example_index]
-    st.sidebar.write(example)
+    with st.form("example_form"):
```

Comment:
This also got reverted.  It should be a slider now.

Reply:
Seems fixed.